### ISSUE TITLE
Index file calculation

### ISSUE THREAD

**Issue by mukarramtahir** [ahasfura/parallel_preseq#1]
Is it feasible (considering the time to compute the index and the filesize of the resulting index) to pre-compute index files for all the BAM files we will be processing? 

Index file is calculated using: 

```
samtools index filename.bam
```

**Comment by lhogstrom** [comment]
I tried to make a few index files for various BAMs on the Broad server. I'm running into this problem:
![image](https://cloud.githubusercontent.com/assets/3091474/11449701/b7390b8a-954d-11e5-8612-c7458a1c74dd.png)

It might be a samtools version issue. I'll look into it further.

**Comment by mukarramtahir** [comment]
Yes, the default version of samtools that you install in Ubuntu with apt-get does not fully support indexing. I had to compile from source and install my own version in my home directory before I could use 'samtools index' successfully.
